kafka/manager.go

```diff
@@ -64,18 +71,36 @@ func NewManager(cfg ManagerConfig) (*Manager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("kafka: failed creating kafka client: %w", err)
 	}
-	return &Manager{
-		cfg:    cfg,
-		client: kadm.NewClient(client),
-		tracer: cfg.tracerProvider().Tracer("kafka"),
-	}, nil
+	m := &Manager{
+		cfg:         cfg,
+		client:      client,
+		adminClient: kadm.NewClient(client),
+		tracer:      cfg.tracerProvider().Tracer("kafka"),
+	}
+
+	mp := cfg.meterProvider()
+	meter := mp.Meter("github.com/elastic/apm-queue/kafka")
+	consumerGroupLagMetric, err := meter.Int64ObservableGauge("kafka.consumer_group_lag")
```

Comment:
I think (not sure if that is better) if we are not going to have more than one observable metric, we can probably just use the `metric.WithInt64Callback` option in `meter.Int64ObservableGauge`.

Reply:
I briefly looked at it, but I don't know how to unregister the callback when the manager is closed. I'm going to leave it for now.